### PR TITLE
Feature/funding account

### PR DESCRIPTION
**Pull request by NoahKr**
Implement funding_accounts/find endpoint

### CODE REVIEW

File: src/Model/FundingAccount.php [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+    private $createdAt;
+    /**
+     * @var DateTime
+     */
+    private $updatedAt;
+
+    /**
+     * @return string
+     */
+    public function getId()
```
Comment (kelvinsouer) [inline src/Model/FundingAccount.php:76]: missing return types (whole class)

File: src/Model/FundingAccounts.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    function __construct(array $fundingAccounts, Pagination $pagination)
+    {
+        parent::__construct($pagination);
+        $this->fundingAccounts = $fundingAccounts;
+    }
+
+    /**
+     * @return FundingAccount[]
+     */
+    public function getFundingAccounts()
```
Comment (kelvinsouer) [inline src/Model/FundingAccounts.php:28]: missing return types (whole class)

**Review by kelvinsouer — CHANGES_REQUESTED** [review]

File: src/Model/FundingAccount.php (hunk 1 above)
Comment (NoahKr) [reply src/Model/FundingAccount.php:76]: I agree, but it's convention of this sdk. I think it's php 5.4 compatible.

File: src/Model/FundingAccounts.php (hunk 2 above)
Comment (NoahKr) [reply src/Model/FundingAccounts.php:28]: I agree, but it's convention of this sdk. I think it's php 5.4 compatible.

**Review by kelvinsouer — APPROVED** [review]

**Pull request merged**